pkg/hive/cell/structured.go

```diff
@@ -415,30 +415,39 @@ func (s *StatusNode) allOk() bool {
 	return s.Level == StatusOK
 }
 
-func (s *StatusNode) writeTo(w io.Writer, d int) {
+func toAgeHuman(t time.Time) string {
+	if t.IsZero() {
+		return "n/a"
+	}
+
+	return duration.HumanDuration(time.Since(t))
+}
+
+func (s *StatusNode) writeTo(w io.Writer, r string) {
+	id := s.Name
+	if r != "" {
+		id = r + "." + id
+	}
 	if len(s.SubStatuses) == 0 {
-		fmt.Fprintf(w, "%s%s\t%s\t%s\t%s ago\t(x%d)\n", strings.Repeat("\t", d), s.Name, s.Level, s.Message, time.Since(s.Timestamp), s.Count)
+		fmt.Fprintf(w, "%s: %s, %s, %d, %s, %v\n", id, s.Level, toAgeHuman(s.Timestamp), s.Count, s.Message, s.Err)
 	} else {
-		fmt.Fprintf(w, "%s%s\n", strings.Repeat("\t", d), s.Name)
 		for _, ss := range s.SubStatuses {
-			ss.writeTo(w, d+1)
+			ss.writeTo(w, id)
 		}
 	}
 }
 
-func (s *StatusNode) toString(ident int) string {
+func (s *StatusNode) toString() string {
 	if s == nil {
 		return ""
 	}
-	buf := bytes.NewBuffer(nil)
-	w := tabwriter.NewWriter(buf, 0, 0, 1, ' ', 0)
-	s.writeTo(w, ident)
-	w.Flush()
-	return buf.String()
+	w := bytes.NewBuffer(nil)
+	s.writeTo(w, s.Name)
+	return w.String()
```

Comment:
Nit: `strings.Builder` should be slightly more efficient and it satisfies `io.Writer` too:

```suggestion
	var sb strings.Builder
	s.writeTo(&sb, s.Name)
	return sb.String()
```